config/youtube.config.example.js

```diff
@@ -0,0 +1,4 @@
+// Put your YouTube API keys here!
+var YOUTUBE_API_KEY = 'YOUR KEY SHOULD GO HERE';
+
```

Comment:
Just leaving a test comment here as I learn how to perform reviews in github

Reply:
Thanks Ronny!